### PR TITLE
Update to 0.2.19

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks for this, @jschueller. If possible, I'd like to do a bugfix on the existing package. Please see PR ( https://github.com/conda-forge/openblas-feedstock/pull/12 ) for details. Also would welcome your feedback on it.

**Comment by jakirkham** [comment]
Please rebase on `master`. Also please feel free to add yourself as a maintainer if you are interested.

**Comment by jschueller** [comment]
@jakirkham is this ok to go in ?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -43,3 +43,5 @@ extra:
     - gillins
     - groutr
     - jakirkham
+    - jschueller
+
```
Comment (jakirkham) [inline recipe/meta.yaml]: Could we please remove the extra newline and keep one terminal newline?

**Comment by jakirkham** [comment]
Please rebase on `master` to clean out the re-rendering.

**Comment by jakirkham** [comment]
Also could you please double check and pin anything that doesn't have `openblas` pinned? This is the [current pinning]( https://github.com/conda-forge/conda-forge.github.io/blob/68b2612a37a3a18bbebb04f08dee91e3fffb218e/scripts/pin_the_slow_way.py#L46 ).

**Comment by jschueller** [comment]
yes, it's all good:
https://github.com/search?l=yaml&p=1&q=org%3Aconda-forge+openblas&type=Code

File: recipe/meta.yaml (hunk 1 above)
Comment (jschueller) [inline recipe/meta.yaml]: ok

**Comment by jakirkham** [comment]
FYI @conda-forge/core we are going to bump the OpenBLAS version. Seems that everything is pinned ok ATM. Though there are some very old packages floating around that did not have proper pinning. It is possible they resurface because of this change. Please keep your eyes open for this and either move these incorrectly pinned packages out of the `main` label to `broken` or ask someone else to do it. If you have any concerns, please let us know.

**Comment by jakirkham** [comment]
Need to re-render first. ( https://github.com/conda-forge/openblas-feedstock/pull/16 )

**Comment by jschueller** [comment]
ok

**Comment by jschueller** [comment]
Ready to release this and break stuff :) ?

**Comment by jakirkham** [comment]
Could you please rebase this , @jschueller ? Then I think we should merge.

**Comment by jschueller** [comment]
ok

**Comment by jschueller** [comment]
@jakirkham ok

**Comment by jschueller** [comment]
@jakirkham ok

**Comment by jschueller** [comment]
ping @jakirkham

**Comment by jakirkham** [comment]
Sorry things came up and I dropped the ball on this. Merging...

**Pull request merged**